tests/PHPStan/Rules/Methods/data/method-call-statement-no-side-effects-phpdoc.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace MethodCallStatementNoSideEffects;
+
+class Bzz {
+	function regular(string $a): string {return $a;}
+	/**
+	 * @phpstan-pure
+	 */
+	function pure1(string $a): string {return $a;}
+	/**
+	 * @psalm-pure
+	 */
+	function pure2(string $a): string {return $a;}
+}
+(new Bzz())->regular('test');
```

Comment:
This should be put into an anonymous function so that it's not called with runtime reflection.